website/utils/tables.ts

```diff
@@ -21,18 +21,4 @@ export const getStaticPaths = () => {
   }
 }
 
-export const getTablesData = () => {
-  const tables = glob.globSync('tables/**/*.md')
-  const tablesData = tables.map((file) => {
-    const withoutDir = file.replace('tables/', '')
-    const [plugin, tableFile] = withoutDir.split('/')
-    const table = path.basename(tableFile, '.md')
-    return {
-        plugin,
-        table,
-    }
-  })
-
-  tablesData.sort((a, b) => a.table.localeCompare(b.table));
-  return tablesData
-};
+export const getTablesData = getTablesDataJS;
```

Comment:
`next-sitemap.config.js` does not support TypeScript, so moved this code to JS for re-use